src/components/AlertDialog/dialogUtils.ts

```diff
@@ -0,0 +1,62 @@
+import type React from 'react';
+
+const emailRegex = (value: string) => !/^\S+@\S+\.\S+$/.test(value);
+const phoneRegex = (value: string) =>
+  !/^\d{10}$/.test(value.replace(/\D/g, ''));
+
+export const formFields = {
+  name: '',
+  phoneNumber: '',
+  email: '',
+  body: '',
+};
+
+export interface FormFieldsType {
+  name: string;
+  phoneNumber: string;
+  email: string;
+  body: string;
+}
+
+export const isEveryErrorCleared = (errors: FormFieldsType) =>
+  Object.values(errors).every((error) => !error);
+
+export const isEveryFieldFilled = (formValues: FormFieldsType) =>
+  Object.values(formValues).every((value) => value.trim());
+
+export const validateField = (
+  name: string,
```

Comment:
Let's avoid confusion by naming it `key`:
```suggestion
  key: string,
```

Reply:
I think we can change it to `fullName` but key for the actual name does not make sense

Reply:
Actually your idea is good. I will rename to Key